deeppavlov/configs/odqa/ranker_prod.json

```diff
@@ -0,0 +1,52 @@
+{
+  "dataset_iterator": {
+    "name": "sqlite_iterator",
+    "data_dir": "odqa",
+    "shuffle": false,
+    "data_url": "http://lnsigo.mipt.ru/export/datasets/wikipedia/wiki_full.db"
```

Comment:
This file is 13GB but it can be gzipped to 4.3GB

Reply:
Yes, but this will take too much time to decompress. So the question is, what we want to save for our users, time or traffic?

Reply:
I decompressed it in a minute and a half. Usually it takes more than that to download 8GB.

Reply:
Let's try to save traffic.

Reply:
On my machine it takes much longer to decompress (about 7 mins)
```
2018-04-06 12:14:08.779 INFO in 'deeppavlov.core.data.utils'['utils'] at line 54: Downloading from http://lnsigo.mipt.ru/export/deeppavlov_data/odqa.tar.gz to /media/olga/Data/projects/iPavlov/DeepPavlov/download/odqa.tar.gz
100%|██████████████████████████████████████████████████████████████████████████████████| 4.53G/4.53G [01:23<00:00, 54.0MB/s]
2018-04-06 12:15:32.681 INFO in 'deeppavlov.core.data.utils'['utils'] at line 120: Decompressing...
2018-04-06 12:21:49.946 INFO in 'deeppavlov.core.data.utils'['utils'] at line 118: Extracting /media/olga/Data/projects/iPavlov/DeepPavlov/download/seq2seq_go_bot.tar.gz archive into /media/olga/Data/projects/iPavlov/DeepPavlov/download

```

Reply:
Compressed.